packages/@aws-cdk/pipelines/lib/stage.ts

```diff
@@ -371,6 +372,15 @@ export interface AddStageOptions {
    * @default false
    */
   readonly manualApprovals?: boolean;
+  /**
+   * Add room for extra sequetial intermediary actions
+   *
+   * This increases the executeRunOrder of the execute change set action with
+   * the value passed.
+   *
+   * @default 0
+   */
+  readonly roomForIntermediaryActions?: number;
```

Comment:
I'm also not too happy with the name, but I'm struggling to come up with a better one.

How about:

* `prepareExecuteDistance`
* `runOrderSpace`
* `runOrderDistance`
* `extraRunOrderSpace`   // leaning most towards this
* `runOrderIncrement` (default: 1, cannot be <1)

Also, can add to the comment block why you might want to use this? Something like: `You can use this to make extra room in the runOrder sequence between the changeset 'prepare' and 'execute' actions and insert your own actions there`.

Heads up: this API is going to change in the somewhat near future, because I'm *very* unhappy that users are being forced to deal with `runOrders` at all. We don't know what the final form will be though.

Reply:
Happy to change it and I agree `extraRunOrderSpace` better illustrates the intention of this work